src/routes/reference/solid-router/data-apis/action.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: action
+---
+
+Actions are data mutations that can trigger invalidations and further routing. A list of prebuilt response helpers can be found below.
+
+```jsx
+import { action, revalidate, redirect } from "@solidjs/router"
+
+// anywhere
+const myAction = action(async (data) => {
+  await doMutation(data);
+  throw redirect("/", { revalidate: getUser.keyFor(data.id) }); // throw a response to do a redirect
+});
+
+// in component
+<form action={myAction} method="post" />
+
+//or
+<button type="submit" formaction={myAction}></button>
+```
+
+Actions only work with post requests, so make sure to put `method="post"` on your form.
+
+Sometimes it might be easier to deal with typed data instead of `FormData` and adding additional hidden fields. For that reason Actions have a with method. That works similar to `bind` which applies the arguments in order.
+
```

Comment:
I fixed up this sentence a bit but can you add a link to the `bind` (i believe there's an MDN ref) and can you elaborate on what "order" means (ie. in order of being added to `with`)